askama_derive/src/input.rs

```diff
@@ -181,7 +181,7 @@ impl Default for Print {
 }
 
 #[doc(hidden)]
```

Comment:
Not needed now that the function is private.

Reply:
Removed.